src/routes/books.ts

```diff
@@ -6,6 +6,8 @@ import { BooksStore } from '../books-store';
 import { HTTP } from './http';
 import { NotificationService } from '../notification-service';
 
+const ISBN_CHECK_DELAY = 2_000;
```

Comment:
Ich hätte jetzt spontan 500-1000 ms genommen, damit es sich nicht zu lange anfühlt. Was denkst Du? @d-koppenhagen

Reply:
Dann sieht man es aber auch nicht so schnell.
Man gibt ja in der Regel eh nach der ISBN noch andere Dinge ein...
Ich finde 2 Sekunden schon ganz gut.
Ich glaube der Animationszyklus (pulsieren) dauert eine Sekunde. Dann sieht man es zweimal pulsieren auch wenn die API direkt antworten würde

Reply:
alright